stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/MessageInputViewModel.kt

```diff
@@ -101,8 +103,27 @@ public class MessageInputViewModel @JvmOverloads constructor(
         activeThread.postValue(null)
     }
 
+    /**
+     * Stores the selected mention to a [Set], user for populating the mentioned user IDs.
+     *
+     * @param user The selected user to mention.
+     */
+    public fun selectMention(user: User) {
+        this.selectedMentions += user
+    }
+
+    /**
+     * Sends a regular message to the channel.
+     *
+     * @param messageText The current message text.
+     * @param messageTransformer Transformer that applies custom changes to the message, before being sent.
+     */
     public fun sendMessage(messageText: String, messageTransformer: Message.() -> Unit = { }) {
```

Comment:
I believe the `editMessage()` function also needs these changes, as they could have added a mention while editing.

Reply:
Good point! Easy to overlook! :]

I'll add the changes there too!

Reply:
@hustoj2 https://github.com/GetStream/stream-chat-android/pull/3088 This PR will be most probably pushed to main for a patch update! :]

The comment was addressed there, so we'll pull the changes once things are cleared up and merged! :]